lib/ansible/modules/files/replace.py

```diff
@@ -92,63 +96,81 @@
     version_added: "2.4"
 notes:
   - As of Ansible 2.3, the I(dest) option has been changed to I(path) as default, but I(dest) still works as well.
+  - As of Ansible 2.x, the combined use of I(before) and I(after) options was changed significantly. See
```

Comment:
Changed this from `2.5`/`2.6` since I'm unsure what the target version would be, at this point?